mythril/laser/ethereum/svm.py

```diff
@@ -482,18 +513,8 @@ def register_hooks(self, hook_type: str, hook_dict: Dict[str, List[Callable]]):
 
     def register_laser_hooks(self, hook_type: str, hook: Callable):
         """registers the hook with this Laser VM"""
-        if hook_type == "add_world_state":
-            self._add_world_state_hooks.append(hook)
-        elif hook_type == "execute_state":
-            self._execute_state_hooks.append(hook)
-        elif hook_type == "start_sym_exec":
-            self._start_sym_exec_hooks.append(hook)
-        elif hook_type == "stop_sym_exec":
-            self._stop_sym_exec_hooks.append(hook)
-        elif hook_type == "start_sym_trans":
-            self._start_sym_trans_hooks.append(hook)
-        elif hook_type == "stop_sym_trans":
-            self._stop_sym_trans_hooks.append(hook)
+        if hook_type in self.laser_hooks_dict:
```

Comment:
why is this in this pr?

Reply:
I first added a hook for recording the initial account, it looked a bit of an overkill, so reverted that.(anyway this might help)